packages/grafana-ui/src/components/index.ts

```diff
@@ -203,7 +203,7 @@ export { Button, LinkButton, type ButtonVariant, ButtonGroup, type ButtonProps,
 export { ToolbarButton, ToolbarButtonRow } from './ToolbarButton';
 export { ValuePicker } from './ValuePicker/ValuePicker';
 export { fieldMatchersUI } from './MatchersUI/fieldMatchersUI';
-export { Link } from './Link/Link';
+export { TextLink } from './Link/TextLink';
```

Comment:
This will be a breaking change, we would have to be completely confident that no one uses it. Maybe we can ask the team if there is any way we could be 100% sure no one is using this

Reply:
They are using it, that was the question (apart from the external icon when it is inline) I was going to ask in the meeting today (but when I realised I had something to ask everyone was leaving the meeting 😅 )
Should we keep them the name although we are just supporting text as a link? I think if we keep it we just add this fact to the doc.